### ISSUE TITLE
why class was inherited from "\Google\Protobuf\Internal\Message"?

### ISSUE THREAD

**Issue by nevernet** [stanley-cheung/Protobuf-PHP#14]
my request or response classes were inherited from "\Google\Protobuf\Internal\Message", but not inherited from "\DrSlump\Protobuf\Message"

so i got an error when i execute client php file: 

```
Fatal error: Call to undefined method Rpcapi\IDRequest::serialize()
```

and second problem is there is no Client class was generated, i had to write manully:

```
class IDServiceClient extends \Grpc\BaseStub
{
    public function __construct($hostname, $opts)
    {
        parent::__construct($hostname, $opts);
    }

    /**
     * @param IDRequest $argument
     * @param array $metadata
     * @param array $options
     * @return \Grpc\SimpleSurfaceActiveCall
     */
    public function ValidateID(IDRequest $argument, $metadata = array(), $options = array())
    {
        return $this->_simpleRequest('/rpcapi.IDService/ValidateID', $argument, '\RpcApi\IDResponse::deserialize',
            $metadata, $options);
    }
}
```

please advice
thank you!

Daniel

**Comment by nevernet** [comment]
generation command is:

```
./php/vendor/bin/protoc-gen-php.php -i . -o php rpcapi/idcn.proto
```

**Comment by stanley-cheung** [comment]
Can you let me know which version of gRPC you are using? You shouldn't be getting "\Google\Protobuf\Internal\Message" in your generated code yet. If you had gotten that, you may have followed the instruction from the `master` branch, which is not released yet. 

The last released version is 1.0.0, and grpc.io should have the correct documentation with respect to that released version.

**Comment by nevernet** [comment]
@stanley-cheung thanks
